src/pages/mypage/ProfilEdit.tsx

```diff
@@ -0,0 +1,102 @@
+import { useRef, useState } from "react";
+import styles from "@/pages/mypage/ProfileEdit.module.css";
+import Input from "@/components/input/Input";
+import Button from "@/components/button/Button";
+import camera from "/public/camera.svg";
+
+// import { useForm } from "react-hook-form";
+// import Field from "@/components/field/Field";
+// import UniversitySelect from "@/pages/auth/signUp/UniversitySelect";
+// import PositionSelect from "@/pages/auth/signUp/PositionSelect";
+// import { zodResolver } from "@hookform/resolvers/zod";
+// import { signUpFormSchema } from "@/pages/auth/signUp/constants";
+
+const ProfilEdit = () => {
+  const imageInputRef = useRef<HTMLInputElement | null>(null);
+  const [imageURL, setimageURL] = useState<string | null>(null);
+  const MAX_IMAGE_SIZE_BYTES = 10 * 1024 * 1024;
```

Comment:
MAX_IMAGE_SIZE_BYTES는 렌더링 시마다 새로 선언될 필요가 없는 상수이므로, 컴포넌트 외부로 빼는 것이 좋아보입니다.
```suggestion
  const MAX_IMAGE_SIZE_BYTES = 10 * 1024 * 1024;
  const ProfileEdit = ()=>{
```

Reply:
넵 수정해놓겠습니다!

Reply:
둘다 수정 완료했습니다